### PR TITLE
Update CSP for universal nav

### PR DESCRIPTION
**Pull request by TsayAdobe**
Resolves : [MWPW-143715](https://jira.corp.adobe.com/browse/MWPW-143715)
Appswitcher needs resources from adobeccstatic.com.

Before:
- https://stage--dc--adobecom.hlx.page/acrobat/online/jpg-to-pdf

After:
- https://mwpw-143629--dc--adobecom.hlx.page/acrobat/online/jpg-to-pdf

### CODE REVIEW

File: acrobat/scripts/contentSecurityPolicy/dev.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const connectSrc = [
   '14257-chimera-stage.adobeioruntime.net',
   '14257-chimera.adobeioruntime.net',
   '*.adobe.com',
+  '*.adobeccstatic.com',
```
Comment (peller) [inline acrobat/scripts/contentSecurityPolicy/dev.js:13]: do we need the wildcard? how many hosts are possible?

Comment (Blainegunn) [reply acrobat/scripts/contentSecurityPolicy/dev.js:13]: @TsayAdobe do we need the `*`

**Review by Blainegunn — APPROVED** [review]

File: acrobat/scripts/contentSecurityPolicy/dev.js (hunk 1 above)
Comment (JFernandezAdobe) [reply acrobat/scripts/contentSecurityPolicy/dev.js:13]: These are the hosts I found in [UniversalNav.js](https://stage.adobeccstatic.com/unav/1.0/UniversalNav.js):

- https://prod.adobeccstatic.com
- https://dev.adobeccstatic.com
- https://stage.adobeccstatic.com

**Review by JFernandezAdobe — APPROVED** [review]

File: acrobat/scripts/contentSecurityPolicy/dev.js (hunk 1 above)
Comment (peller) [reply acrobat/scripts/contentSecurityPolicy/dev.js:13]: if that's the case, we should list each of those hosts under the dev/stage/prod sections, respectively.  no wildcard.

Comment (TsayAdobe) [reply acrobat/scripts/contentSecurityPolicy/dev.js]: Updated with exact hosts

**Comment by peller** [comment]
only prod includes the rule for style-src. Is that intentional?

File: acrobat/scripts/contentSecurityPolicy/dev.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const connectSrc = [
   '14257-chimera-stage.adobeioruntime.net',
   '14257-chimera.adobeioruntime.net',
   '*.adobe.com',
+  'stage.adobeccstatic.com',
```
Comment (peller) [inline acrobat/scripts/contentSecurityPolicy/dev.js:13]: should these be stage.adobeccstatic.com or dev.adobeccstatic.com?

Comment (TsayAdobe) [reply acrobat/scripts/contentSecurityPolicy/dev.js:13]: Our dev env (hlx.page and hlx.live) loads the FEDS stage scripts.

**Review by peller — APPROVED** [review]

**Pull request merged**